src/lib/jsonParseV2/parseNotification.cpp

```diff
@@ -116,6 +116,8 @@ static bool parseNotificationData
 
```

Comment:
Please add an entry to CHANGES_NEXT_RELEASE file regarding the fix in this PR. I suggest something like this:

```
- Fix: memory leak in POST /v2/op/notify operation (#4261)
```

Reply:
Thanks for the suggestion. I have added the entry.